### ISSUE TITLE
remove an eeprom-loading hardcode in initTask 

### ISSUE THREAD

**Issue by pkotamnives** [apollo-lhc/cm_mcu#148]
issue : there is [a hardcode in initTask](https://github.com/apollo-lhc/cm_mcu/blob/master/projects/cm_mcu/InitTask.c#L47-L48) for loading a clock config from eeprom to the CM board, while the detection of a garbage eeprom has been implemented [here](https://github.com/apollo-lhc/cm_mcu/commit/6cce5661bc636db3d1d1b49d365d2d3eba64cda5). 

Maybe I forgot to remove this hardcode accordingly? 

resolve: will try removing this hardcode and check if the detection of garbage eeprom will quit the initialization as expected.

**Comment by pwittich** [comment]
i think I've seen this working -- check one of the boards in the shelf now. I think your code detects the empty eeprom fine

**Comment by pkotamnives** [comment]
Thanks for confirming. Then, I will remove this hardcode and merge its branch to the master, or you could test the new branch before I merge? but it might have to be after I finish verifying a good old branch for programing the clock chips for Alec's task.

**Comment by pwittich** [comment]
what I meant to say was it works in its current configuration?

**Comment by pkotamnives** [comment]
Yes, I just tried running a command `loadclock 0` and  `loadclock 1` with the master branch code, and the command works just fine. However, once that two lines of hardcode in initTask.c got removed, even `loadclock 0` command no longer works and it got stuck at this https://github.com/apollo-lhc/cm_mcu/blob/master/projects/cm_mcu/LocalTasks.c#L1303 infinitely. I think there might be some things under the rug to investigate.
